app/app.go

```diff
@@ -699,14 +724,29 @@ func NewStrideApp(
 	icacallbacksStack = icaoracle.NewIBCMiddleware(icacallbacksStack, app.ICAOracleKeeper)
 	icacallbacksStack = icacontroller.NewIBCMiddleware(icacallbacksStack, app.ICAControllerKeeper)
 
+	// Create Transfer Stack
+	// SendPacket, since it is originating from the application to core IBC:
+	// transferKeeper.SendPacket -> packetforward.SendPacket -> channel.SendPacket
+
+	// RecvPacket, message that originates from core IBC and goes down to app, the flow is the other way
+	// channel.RecvPacket -> packetforward.OnRecvPacket -> transfer.OnRecvPacket
+
 	// Stack three contains
-	// - IBC
+	// - core ibc
 	// - autopilot
 	// - records
 	// - ratelimit
+	// - pfm
```

Comment:
what's your rationale for the ordering here (not objecting, just curious)? Also we may want to look through the error cases in pfm and confirm there's no scenario where it short circuits before getting to the rest of the stack

Reply:
it feels conceptually closest to ibc transfers, which is why I put it at the bottom (so no strong reason - we can put it elsewhere)